README.md

```diff
@@ -1,6 +1,21 @@
-# curriculum-cr-training-mobile-menu
+# Portfolio
+ 
+## Built With
 
```

Comment:
- [ ] Part of having professional documentation is to have a snapshot of the project to give the viewer a quick idea about your technical skills of implementing a design so please pick a picture of the portfolio and make sure it demonstrates your technical ability